aws-django-voting-app/__main__.py

```diff
@@ -0,0 +1,381 @@
+# Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+import json
+import base64
+import pulumi
+import pulumi_aws as aws
+import pulumi_docker as docker
+import pulumi_mysql as mysql
+
+# Get neccessary settings from the pulumi config
+config = pulumi.Config()
+sql_admin_name = config.require("sql-admin-name")
+sql_admin_password = config.require_secret("sql-admin-password")
+sql_user_name = config.require("sql-user-name")
+sql_user_password = config.require_secret("sql-user-password")
+availability_zone = pulumi.Config("aws").get("region")
+
+# Credentials for the Django site administration panel
+django_admin_name = config.require("django-admin-name")
+django_admin_password = config.require_secret("django-admin-password")
+
+# A random 50-character string
+django_secret_key = config.require_secret("django-secret-key")
+
+# The ECS cluster in which our application and databse will run
+app_cluster = aws.ecs.Cluster("app-cluster")
+
+# Creating a VPC and a public subnet
+app_vpc = aws.ec2.Vpc("app-vpc",
+    cidr_block="172.31.0.0/16",
+    enable_dns_hostnames=True)
+
+app_vpc_subnet = aws.ec2.Subnet("app-vpc-subnet",
+    cidr_block="172.31.0.0/20",
+    availability_zone=availability_zone + "a",
+    vpc_id=app_vpc)
+
+# Creating a gateway to the web for the VPC
+app_gateway = aws.ec2.InternetGateway("app-gateway",
+    vpc_id=app_vpc.id)
+
+app_routetable = aws.ec2.RouteTable("app-routetable",
+    routes=[
+        {
+            "cidr_block": "0.0.0.0/0",
+            "gateway_id": app_gateway.id,
+        }
+    ],
+    vpc_id=app_vpc.id)
+
+# Associating our gateway with our VPC, to allow our app to communicate with the greater internet
+app_routetable_association = aws.ec2.MainRouteTableAssociation("app_routetable_association",
+    route_table_id=app_routetable.id,
+    vpc_id=app_vpc)
+
+# Creating a Security Group that restricts incoming traffic to HTTP
+app_security_group = aws.ec2.SecurityGroup("security-group",
+	vpc_id=app_vpc.id,
+	description="Enables HTTP access",
+    ingress=[{
+		'protocol': 'tcp',
+		'from_port': 0,
+		'to_port': 65535,
+		'cidr_blocks': ['0.0.0.0/0'],
+    }],
+    egress=[{
+		'protocol': '-1',
+		'from_port': 0,
+		'to_port': 0,
+		'cidr_blocks': ['0.0.0.0/0'],
+    }])
```

Comment:
Use input classes and use `"` rather than `'` to be consistent with quotes used throughout the rest of this file.

```suggestion
    ingress=[aws.ec2.SecurityGroupIngressArgs(
		protocol="tcp",
		from_port=0,
		to_port=65535,
		cidr_blocks=["0.0.0.0/0"],
    )],
    egress=[aws.ec2.SecurityGroupEgressArgs(
		protocol="-1",
		from_port=0,
		to_port=0,
		cidr_blocks=["0.0.0.0/0"],
    )])
```